galaxy_ng/app/api/ui_v2/views.py

```diff
@@ -0,0 +1,173 @@
+from rest_framework import viewsets
+from django_filters.rest_framework import DjangoFilterBackend
+from django.http import JsonResponse
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework.exceptions import ValidationError
+from rest_framework.decorators import action
+
+from ansible_base.rest_pagination.default_paginator import DefaultPaginator
+
+from .filters import UserViewFilter
+from .filters import GroupViewFilter
+from .filters import OrganizationFilter
+from .filters import TeamFilter
+from .serializers import UserSerializer
+from .serializers import GroupSerializer
+from .serializers import OrganizationSerializer
+from .serializers import TeamSerializer
+
+from galaxy_ng.app.models.auth import User
+from galaxy_ng.app.models.auth import Group
+from galaxy_ng.app.models.organization import Organization
+from galaxy_ng.app.models.organization import Team
+
+
+def version_view(request):
+    data = {
+        "version": "2024-08-05T18:00:00"
```

Comment:
Is this going to be dynamically fetched?

Reply:
Nah, i'm going to remove it. I added it as a debug thing last week when the UI was having issues getting the right DAB version.